src/components/Router/index.jsx

```diff
@@ -6,19 +6,23 @@ import App from '../App';
 import { CONFIG } from '../../config';
 import validCombination from '../../utils/validCombination';
 
+
 const Router = () => (
   <BrowserRouter>
     <Switch>
       <Redirect from="/win10Laptops/:benchmark" to="/win10ref2017/:benchmark" />
+      <Redirect exact from="/:platform/tp6-:site" to="/:platform/page-load/tp6-:site" />
```

Comment:
Mobile page load suites start with 'tp6m-' and are currently redirecting to '/win10/benchmarks/overview' can we include these in the redirect?

Reply:
I've added an additional redirect to cover these suites